recipes/oboe/all/patches/1.7.0-0001-fix-cmake.patch

```diff
@@ -12,7 +12,7 @@
          -Ofast
 -        "$<$<CONFIG:DEBUG>:-Werror>")
 +)
-+target_compile_features(oboe PUBLIC cxx_std_17)
```

Comment:
Why is it removed?

This workaround in conanfile is worse:

```
        if not valid_min_cppstd(self, self._min_cppstd):
            tc.variables["CMAKE_CXX_STANDARD"] = self._min_cppstd
```

Reply:
This workaround:
- hardcodes C++17 when compiler.cppstd is not set in profile while target_compile_features() is smarter and can fallback to something closer to default compiler version as soon as it honors min C++ version.
- makes conanfile more complex than it needs to be

Moreover the way to go in CCI recipes is to patch and submit fixes upstream to avoid workarounds in conanfile (I have to admit I didn't have the time to submit this one).